fly_manager.py

```diff
@@ -3,37 +3,42 @@
 import os
 from set_env import EnvironmentVarSetting
 import sys
+from dotenv import dotenv_values
+
+import os
+from dotenv import load_dotenv
+from pathlib import Path
```

Comment:
we can import load_dotenv and dotenv_values in single import no need to call import multiple times.